reactive/flume_hdfs.py

```diff
@@ -0,0 +1,42 @@
+from charms.reactive import when, when_not
+from charms.reactive import set_state, remove_state
+from charmhelpers.core import hookenv
+from charms.flume import Flume
+from charms.hadoop import get_dist_config
```

Comment:
Technically, since this charm is pulling in hadoop-client, either should work.  And the version from hadoop-client handles the hadoop-client layer options as well, so it's probably better to use that, if available.